docs/sql/sql-full-text.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: default
+title: Full-Text Search
+parent: SQL
+nav_order: 2
+---
+
+# Full-text search
+
+Use SQL commands for full-text search. For more information about full-text queries in Elasticsearch, see [Full-text queries](../../elasticsearch/full-text/).
```

Comment:
Probably worth mentioning here that SQL has a subset of the full-text queries available in Elasticsearch.

Reply:
Done.